conf/upf.jsonc

```diff
@@ -155,7 +155,12 @@
     "cpiface": {
         "peers": ["148.162.12.214"],
         // [Optional] Below parameters
-        "dnn": "internet",
+        "dnn_list": [
+            {
+                "dnn": "internet",
+                "ue_ip_pool": "10.250.0.0/16"
```

Comment:
Also, can we please update this to use a private IP range/pool?

Reply:
@gab-arrobo  Isn't it like 10.0.0.0/8 is complete Class A private IP address range. So 10.250.0.0/16 is part of private IP address range.